### PR TITLE
D8CORE-1289: Update homepage banner default content

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Change default content to support mo' betta user workflow

# Needed By (Date)
- v1.0.0

# Criticality
- How critical is this PR on a 1-10 scale? 7/10
- It's the front door of our product

# Steps to Test

1. Check out this branch
2. Install this profile
3. Check that banner looks like this:
2020-02-12-101639-screenshot
4. Check that the hyperlink sends you to SAML login

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- JIRA ticket: D8CORE-1289

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json [hunk 1]
```diff
@@ -79,9 +79,9 @@
     ],
     "su_banner_body": [
         {
-            "value": "<p>To edit this page, add \"\/user\" to the end of your url, and login to Drupal<\/p>\r\n",
+            "value": "<p>To edit this page, <a href=\"\/saml_login\">log in with your SUNetID<\/a>. After you log in, return to the homepage, and use it as a template to begin building your site.<\/p>\r\n",
```
Comment (sherakama) [inline content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json]: Shouldn't this be `/sso/login`?

Comment (pookmish) [reply content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json]: i don't think it matters much in the end. we (john) can change later if we see a need.

Comment (jbickar) [reply content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json]: Did the `/sso/login` work get merged yet?

Comment (jbickar) [reply content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json]: I would prefer `/sso/login`, but I wasn't sure if that was in yet.

Comment (sherakama) [reply content/paragraph/5395f779-e884-4429-a533-a6712bd4eb32.json]: https://github.com/SU-SWS/stanford_ssp/releases/tag/8.1.2

**Pull request merged**